### ISSUE TITLE
test_hypergraph fails

### ISSUE THREAD

**Issue by erelsgl** [pnnl/HyperNetX#160]
**Describe the bug**
When submitting a PR, the github actions return an unrelated error. See [here](https://github.com/rotshira/HyperNetX/actions/runs/10178283715/job/28151598810):

**To Reproduce**
1. Submit any PR.
2. Go to "actions" and look at "macos-latest, 3.10" test.
3. See error.

**Expected behavior**
Test should pass.

**Screenshots**
![image](https://github.com/user-attachments/assets/127dedf9-1292-4b22-84e6-60f8e75de2b6)

**Environment (please complete the following information):**
 - MacOS on GIthub Actions.
 - Python Version: 3.10 on Github Actions.

**Comment by brendapraggastis** [comment]
Yep. We are on it. hotfix will get pushed today to version 2.3.6. Thanks!
